integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTwoDomainsLoadBalancers.java

```diff
@@ -460,36 +503,76 @@ public void testTraefikHostHttpsRoutingAcrossDomains() {
    * Accesses the clusterview application deployed in the WebLogic cluster through Voyager loadbalancer and verifies it
    * is correctly routed to the specific domain cluster identified by the -H host header.
    */
-  @Order(7)
+  @Order(8)
   @Test
-  @DisplayName("Loadbalance WebLogic cluster traffic through Voyager loadbalancer tcp channel")
+  @DisplayName("Loadbalance WebLogic cluster traffic through Voyager loadbalancer host routing")
   public void testVoyagerHostHttpRoutingAcrossDomains() {
 
     // verify load balancing works when 2 domains are running in the same namespace
-    logger.info("Verifying http traffic");
+    logger.info("Verifying Voyager http routing");
+    for (String domainUid : domainUids) {
+      String ingressName = domainUid + "-voyager-host-routing";
+      verifyClusterLoadbalancing(domainUid, domainUid + "." + defaultNamespace + ".voyager.nonssl.test",
+          "http", getVoyagerLbNodePort(ingressName, "tcp-80"),
+          replicaCount, true, "");
+    }
+  }
+
+  /**
+   * Test verifies multiple WebLogic domains can be loadbalanced by Voyager loadbalancer with tls based routing rules.
+   * Accesses the clusterview application deployed in the WebLogic cluster through Voyager loadbalancer and verifies it
+   * is correctly routed to the specific domain cluster identified by the -H host header.
```

Comment:
Do we need to provide  host header for path routing

Reply:
Those are for host https routing.